tests/rewards/test_reward_nets.py

```diff
@@ -113,11 +169,29 @@ def test_init_no_crash(
         )
 
 
+@pytest.mark.parametrize("env_name", IMAGE_ENVS)
+@pytest.mark.parametrize("reward_net_cls", MAKE_IMAGE_REWARD_NET)
+@pytest.mark.parametrize("reward_net_kwargs", REWARD_NET_KWARGS)
+@pytest.mark.parametrize("normalize_output_layer", NORMALIZE_OUTPUT_LAYER)
```

Comment:
Normalizing *outputs* on the other hand still definitely makes sense for image-based reward network, so glad we're testing this.